tests/LanguageAddTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Unish;
+
+use Drupal\Core\Language\Language;
+use Drupal\language\Entity\ConfigurableLanguage;
+use Webmozart\PathUtil\Path;
+
+/**
+ *  @group slow
+ *  @group pm
+ */
+class LanguageAddCase extends CommandUnishTestCase
+{
+
+    public function testLanguageInfo()
+    {
+        $sites = $this->setUpDrupal(1, true);
+        $uri = key($sites);
```

Comment:
We now default uri to 'dev' so no need to set it anymore. And I think 'root' gets autodiscovered as well. Omit them both.